app/views/ProjectEdit/Users/index.tsx

```diff
@@ -21,21 +50,25 @@ function Users(props: Props) {
         activeUserId,
```

Comment:
Do we not need activeUserId anymore?

Reply:
Yes it has been used to disable a particular column in a table conditionally.